prExplorer/API/obj/project.assets.json

```diff
@@ -2021,8 +2021,8 @@
         "net8.0": {
           "targetAlias": "net8.0",
           "projectReferences": {
-            "c:\\Users\\Ermat\\Desktop\\NC Github\\NC\\prExplorer\\Application\\Application.csproj": {
```

Comment:
Unfortunately, all of those files were added again... you need to merge your `mobx` branch first where these files were deleted, then it should be OK. You will need to rebase this branch before merging this PR - let me know if you have problems with that